src/components/card-container/card-container.js

```diff
@@ -26,7 +26,7 @@ class CardContainer extends React.PureComponent {
     return (
       <div>
         <a href={props.path} className="unprose mb18 block color-blue-on-hover">
-          <h2 className="txt-bold" id={categoryID}>
+          <h2 className="txt-bold unprose" id={categoryID}>
```

Comment:
Note: Due to the arranging of the PageLayout content component, needed to add `unprose` here to prevent these headings from getting styled.